package.json

```diff
@@ -17,7 +17,7 @@
     "start": "npm run develop",
     "serve": "gatsby serve",
     "lint": "eslint --fix .",
-    "checklint": "eslint .",
+    "checklint": "eslint --fix .",
```

Comment:
@debo19 I guess this was unintentional?

Reply:
@Nikhil-Ladha it was intentional. While commiting my changes, the space/indentation errors weren't fixing on its own. Adding `--fix` here worked for me.

Reply:
Well, that's the thing :)

Even if it would have been fixed it wouldn't have got added to your `staged` files automatically, and you will possibly forget adding it if it does fix it automatically. Hence, the `--fix` option was removed to give the chance to the author to always add the fixed changes after the linter fixes it for you.

Reply:
To fix the eslint issues (if fixable automatically) one can run `npm run lint`.

Reply:
I have reverted it in my latest PR for sitemap #3013

Reply:
Ohhh got it. I thought it'll add it automatically(and clearly I forgot about your PR). Apologies 😬

Reply:
No worries 😄